pkg/pool-weighted/test/IndexPool.test.ts

```diff
@@ -125,11 +124,111 @@ describe('IndexPool', function () {
         pool = await WeightedPool.create(params);
       });
 
-
       it('swaps are blocked', async () => {
         await expect(pool.swapGivenIn({ in: 1, out: 0, amount: fp(0.1) })).to.be.revertedWith('MAX_IN_RATIO');
       });
     });
+  });
+
+  describe('#reweighTokens', () => {
+    sharedBeforeEach('deploy pool', async () => {
```

Comment:
I think we should have for actual succesful execution of `reweighTokens` and `reindexTokens`, or is it too early?

Reply:
Not sure I understand your suggestion. You mean add a test to check that nothing reverts if the parameters are correct? If that is what you mean I would rather wait for future PRs where we implement actual functionalities that we can test.

Reply:
Yep, that's exactly the thing I was referring to, sounds ledgit...